docs/installation.rst

```diff
@@ -264,23 +270,35 @@ For example:
       # 1. Change the command to run
         command: yarn run debug
         volumes:
-          - "./config/config.json:/home/bookbrainz/bookbrainz-site/config/config.json:ro"
       # 2. Mount the src directory
           - "./src:/home/bookbrainz/bookbrainz-site/src"
+
+Ideally you save this new Compose file inside the ``local/`` directory, which will be ignored by git, for example as ``local/docker-compose.live-reload.yml``.
+
+Now you have to explicitly tell Docker-Compose which Compose files it should read when the ``./develop.sh`` script is run.
+In addition to the default ``docker-compose.live-reload.yml`` we also want Compose to read our custom file with the overrides.
```

Comment:
Should this read "the default docker-compose.yml" instead of the live-reload override file?

Reply:
You're right of course, I vaguely remember having fixed that already, but I didn't find the changes in one of my two local clones. So I had probably only _intended_ to fix it after copy-pasting the markup from above (entering two consecutive backticks with a keyboard is a PITA, let alone having to do it twice because VS Code's feature to wrap selected text automatically did not work for this kind of quotes).
Anyway, I've fixed this line now and rebased onto the latest master.